docs/how_to_write_custom_backend.rst

```diff
@@ -0,0 +1,193 @@
+How to write custom backend?
+============================
+
+You can write your own backend for any service of your choice, instead of ``Twilio`` and ``Nexmo``, which are already provided by ``django-phone-verify``.
```

Comment:
```suggestion
In case you want to use anything other than the provided backends (``backends.nexmo.NexmoBackend`` and ``backends.twilio.TwilioBackend``), you can write a custom backend by extending ``backends.base.BaseBackend`` as shown below.
```